patches/015-fix-crashdumpallthreads.patch

```diff
@@ -0,0 +1,61 @@
+From 8d2ec6b3704753e17dbafbc2407d7f4c808fd57f Mon Sep 17 00:00:00 2001
```

Comment:
Can you remove this section of the patch and just include the diff please.

Reply:
done